client/src/endpoint/mod.rs

```diff
@@ -17,3 +20,90 @@ pub trait EndPoint {
     /// Converts the implementing struct into an http request.
     fn into_request(self, host: &str) -> Result<http::Request<Self::RequestBody>>;
 }
+
+/// The order to return results in.
+#[derive(Debug)]
+pub enum Order {
+    /// Order the results ascending
+    Asc,
+    /// Order the results descending
+    Desc,
+}
+
+impl Order {
+    pub(crate) fn to_param(&self) -> String {
+        match *self {
+            Order::Asc => "asc".to_string(),
+            Order::Desc => "desc".to_string(),
+        }
+    }
+}
+
+/// A struct that represents a set of records that comes from an embedded resource.
+#[derive(Debug)]
```

Comment:
Well done.  Maybe a short comment about how resources from the stellar API are nested under _embedded and records. 🤷‍♂️

Reply:
I might add that comment to the embedded struct/records but not to the records abstraction.